### DISCUSSION TITLE
All vaues showing zero

### DISCUSSION THREAD

**Discussion by amitjindal** [umami-software/umami#898] [Q&A] [answered]
Hi, I upgraded to latest code and for some reason all my metrics are showing as zero values. This was happening from previous version also.
I am running this inside docker. I am not sure what to check. The logs also did not contain anything wrong.

How can I resolve this? Where do I look?

```
root@app02:~# docker logs  umami.web.1
yarn run v1.22.15
$ next start
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
```
![2021-12-15_23-40-04](https://user-images.githubusercontent.com/3275983/146241824-55d0b873-c7c0-43bf-8864-08191a6e29fd.jpg)

Thanks
Amit

**Comment by amitjindal** [comment]
Hi, Anyone can help?
@mikecao If you can point me in right direction I will try to solve it myself.

**Comment by mikecao** [comment, marked as answer]
You can try cloning the repo and running in dev mode. If you connect to your current database, it should work the same, Then, set `LOG_QUERY = 1` in your .env file and you should see the SQL queries printed to the log output. Run those queries to see what data is being returned.

**Reply by amitjindal** [reply]
I tried this. But didn't seem to help. All queries were correct.
I also noticed that the API calls (XHR) in browser were returning correct data. So tried to use this from another browser and found that an extension in Firefox was causing a display issue. The extension causing this was "JavaScript Restrictor". Only display of values was affected. The graphs were still correct.

This turned out to be a just a non-issue.

Thanks for the help.
